bundles/org.openhab.ui.iconset.classic/README.sh

```diff
@@ -42,7 +42,7 @@ cat <<EOF >> "$READMEMD"
 {% for category in placesCategoryNames %}
   {% assign iconSrc = base | append: "/img/icon_no_category.png" %}
   {% if allIconsWithoutExtension contains category %}
-    {% assign iconSrc = "icons/" | append: category | append: ".png" %}
+    {% assign iconSrc = "icons/" | append: category | append: ".svg" %}
   {% endif %}
   <figure>
     <img src="{{iconSrc}}" alt="{{category}}" title="{{category}}">
```

Comment:
Maybe a size should be added here. Before the PNG icons had a fixed size, but SVG doesn't and it should be avoided to show them too big in the docs.

Reply:
Yes, you're right.

Reply:
What size should I set?

Reply:
The size of the png icons should be a good choice as the page doesn't change then. Currently the pngs are 64x64 pixel.

Reply:
https://www.openhab.org/docs/configuration/iconsets/classic/

In case, this stuff is used to build this page, yes, the icons are currently displayed with 64x64 as size.